src/App.vue

```diff
@@ -1,16 +1,18 @@
 <template>
   <div class="app">
-    <Header/>
+    <TheHeader/>
+
+    <router-view/>
   </div>
 </template>
 
 <script>
-import Header from "./components/Header";
+import TheHeader from "./components/TheHeader";
```

Comment:
I couldn't start an application because of the component name error. It should be a multi-word name. And by the Vue style guide, singleton components should be prefixed with "The"